src/MockData/cities.json

```
@@ -0,0 +1,962 @@
+[
+  {
+    "id": 0,
+    "name": "Tripoli",
+    "code": "tri7",
+    "description": "Tripoli is the largest city in northern Lebanon and the second-largest city in the country",
+    "lat": 34.46664758010337,
+    "lng": 35.831724876807975,
+    "address_coordinates": [34.46664758010337, 35.831724876807975],
+    "country": "Lebanon",
+    "iso2": "LB",
+    "admin_name": "Liban-Nord",
+    "capital": "admin",
+    "population": "530000",
+    "population_proper": "192572",
+    "image-array": [
```

this should be `images`
```suggestion
    "images": [
```